packages/web/public/static/locales/de/settings.json

```diff
@@ -101,6 +101,14 @@
       "passwordChangeSuccess": "Das Passwort wurde erfolgreich geändert.",
```

Comment:
Hi @LindasLinguas ! Do you think you could translate these new notification settings strings for us? 😀 Vielen Dank!